src/tasks/cleanup.cr

```diff
@@ -0,0 +1,51 @@
+require "json"
+require "pg-orm"
+
+module PlaceOS::Tasks::CleanUp
+  extend self
+  Log = ::Log.for(self)
+
+  def cleanup(config : String, host : String, port : Int32, db : String, user : String, password : String)
+    Log.info do
+      Log.context.set({host: host, port: port, db: db, config: config})
+      "cleaning old records"
+    end
+
+    unless File.exists?(config)
+      Log.error { "config file `#{config}` not found" }
+      exit(1)
+    end
+
+    PgORM::Database.configure do |settings|
+      settings.host = host
+      settings.port = port
+      settings.db = db
+      settings.user = user
+      settings.password = password
+    end
+
+    tables = TableConfig.from_json(File.read(config))
+    tables.each do |table|
+      sql = %(
+            DELETE FROM #{table.table} WHERE updated_at < NOW() - INTERVAL '#{table.interval}'
```

Comment:
this would delete configuration... We only want to clean up items that are no longer relevant